sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala

```diff
@@ -73,10 +73,35 @@ class AllExecutionsPageSuite extends SharedSparkSession with BeforeAndAfter {
     map.put("failed.sort", Array("duration"))
     when(request.getParameterMap()).thenReturn(map)
     val html = renderSQLPage(request, tab, statusStore).toString().toLowerCase(Locale.ROOT)
-    assert(!html.contains("IllegalArgumentException"))
+    assert(!html.contains("illegalargumentexception"))
```

Comment:
Shouldn't it be `IllegalArgumentException`?

Reply:
@shahidki31, No it should be in lowercase because if you see below code there they are using .toLowerCase(Locale.ROOT) which will convert complete html contain to lowercase. 

val html = renderSQLPage(request, tab, statusStore).toString().toLowerCase(Locale.ROOT)

Reply:
Nice catch!